src/v2/wallets.js

```diff
@@ -131,6 +131,10 @@ Wallets.prototype.add = function(params, callback) {
     walletParams.rootPub = params.rootPub;
   }
 
+  if (params.rootPrivateKey) {
```

Comment:
Please add a comment describing our reasons for doing this.
1. The private key required for setting up multi-sig
2. The private key is disabled once the wallet gets funded and multi-sig is enabled on it

I added a comment to the PR with more information